app/assets/stylesheets/_base.scss

```diff
@@ -67,3 +67,15 @@ a, button {
     outline: 0;
   }
 }
+
+.row {
+  padding-left: 15px;
```

Comment:
There is a grid.scss file. Maybe it would be better to move this code to that file